src/core/pointcloud/qgscopcpointcloudindex.cpp

```diff
@@ -432,4 +362,34 @@ void QgsCopcPointCloudIndex::copyCommonProperties( QgsCopcPointCloudIndex *desti
   destination->mLazInfo.reset( new QgsLazInfo( *mLazInfo ) );
 }
 
+QByteArray QgsCopcPointCloudIndex::fetchCopcStatisticsEvlrData()
+{
+  uint64_t offset = mLazInfo->firstEvlrOffset();
+  uint32_t evlrCount = mLazInfo->evlrCount();
+
+  QByteArray statisticsEvlrData;
+
+  for ( uint32_t i = 0; i < evlrCount; ++i )
+  {
+    lazperf::evlr_header header;
+    mCopcFile.seekg( offset );
+    char buffer[60];
+    mCopcFile.read( buffer, 60 );
+    header.fill( buffer, 60 );
+    QByteArray evlrData( header.data_length, Qt::Initialization::Uninitialized );
+    mCopcFile.read( evlrData.data(), header.data_length );
```

Comment:
Let's avoid reading evlrs that we don't need - just skip through them and only read "our" evlr to QByteArray